common/translations/lang/ko.json

```diff
@@ -249,406 +214,456 @@
+    "DISABLED": "비활성화",
+    "LEDGER_TIMEOUT": "요청한 시간이 만료되었습니다",
+    "LEDGER_WRONG_APP": "기기에 대해 잘못된 어플리케이션이 선택됨",
+    "LEDGER_LOCKED": "Ledger 기기가 잠김",
+    "TREZOR_REFERAL": "TREZOR 구매",
+    "KEEPKEY_REFERRAL": "Keepkey 구매",
+    "STEELY_REFERRAL": "Steely 구매",
+    "ETHERCARD_REFERAL": "ether.card 구매",
+    "DISCLAIMER": "면책사항",
+    "INPUT_CONFIRM_PASSWORD_LABEL": "비밀번호 확인",
+    "INPUT_PASSWORD_LABEL": "비밀번호",
+    "INPUT_USERNAME_LABEL": "사용자이름",
+    "INPUT_PASSWORD_PLACEHOLDER": "비밀번호는 반드시 최소 $pass_length 개수의 문자로 구성되어야 합니다.",
+    "GENERATE_KEYSTORE_TITLE": "키스토어 파일 생성하기",
+    "GENERATE_KEYSTORE_ACTION": "키스토어 파일 생성",
+    "GENERATE_KEYSTORE_FAILED": "키스토어 생성이 실패하거나 유효하지 않습니다. 자금을 잃는 것을 방지하기 위해서 망가졌을 가능성이 있는 키스토어 파일을 제공하지는 않습니다. 페이지를 다시 리로딩하거나 다른 브라우저를 사용하여 다시 시도하시기 바랍니다.",
+    "WEAK_PASSWORD": "비밀번호가 충분히 강력하지 않습니다.",
+    "INVALID_PASSWORD": "잘못된 비밀번호",
+    "GENERATE_MNEMONIC_TITLE": "Mnemonic 구문 생성",
+    "GENERATE_THING": "$thing 생성",
+    "REGENERATE_MNEMONIC": "구문 재생성",
+    "CONFIRM_MNEMONIC": "구문 확인",
+    "REVEAL_NEXT_MNEMONIC": "다음 단어 보기",
+    "MNEMONIC_CHOOSE_ADDR": "주소 선택",
+    "MNEMONIC_DESCRIPTION_1": "단어를 순서대로 클릭하세요. 다음 단어를 잊어버리셨다면 '다음 단어 보기' 버튼을 눌러주세요.",
+    "MNEMONIC_DESCRIPTION_2": "단어를 써놓으세요. 단어를 클립보드로 복사하거나 온라인으로 저장하지 마세요.",
+    "MODAL_BACK": "뒤로",
+    "WALLET_UNLOCKING": "잠금해제...",
+    "HELP_ARTICLE_1": "Nano S 지갑으로 MyCrypto 사용하기",
+    "DECRYPT_PROMPT_SELECT_ADDRESS": "주소 선택",
+    "DECRYPT_DROPDOWN_LABEL": "주소",
+    "ACTION_1": "확인",
+    "ACTION_2": "취소",
+    "ACTION_3": "잠금해제",
+    "ACTION_4": "뒤로",
+    "ACTION_5": "더보기",
+    "ACTION_6": "다음",
+    "ACTION_7": "로그아웃",
+    "ACTION_8": "상세",
+    "ACTION_9": "사용가능 여부 확인",
+    "ACTION_10": "완료",
+    "ACTION_11": "전송",
+    "ACTION_12": "다운로드",
+    "ACTION_13": "$thing 다운로드",
+    "ACTION_14": "계속",
+    "DOWNLOAD_PHONE_APP": "$os용 다운로드",
+    "CONFIRM_HARDWARE_WALLET_TRANSACTION": "하드웨어 지갑에서 트랜잭션 확인",
+    "WALLET_LOGOUT_MODAL_TITLE": "로그아웃 하려 합니다",
+    "WALLET_LOGOUT_MODAL_DESC": "이 페이지를 벗어나면 로그아웃 됩니다. 계속 하시겠습니까?",
+    "CONFIRM_TX_MODAL_TITLE": "트랜잭션 확인",
+    "CONFIRM_TX_SENDING": "송금 예정",
+    "CONFIRM_TX_FEE": "트랜잭션 수수료",
+    "CONFIRM_TX_TOTAL": "총계",
+    "CONFIRM_TX_FROM": "보내는 곳",
+    "CONFIRM_TX_TO": "받는 곳",
+    "CONFIRM_TX_VIA_CONTRACT": "$unit 컨트랙트를 사용",
+    "MNEMONIC_FINAL_DESCRIPTION": "완료되었습니다. 지갑을 사용할 준비가 되었습니다. 다음의 4단계를 밟기만 하면 지갑에 접근할 수 있습니다.",
+    "CREATE_FINAL_STEP_1": "MyCrypto 열기",
+    "CREATE_FINAL_STEP_2": "보기 & 보내기로 가기",
+    "CREATE_FINAL_STEP_3": "지갑 형식 고르기",
+    "CREATE_FINAL_STEP_4_MNEMONIC": "구문을 입력하세요",
+    "CREATE_FINAL_STEP_4_KEYSTORE": "파일과 비밀번호를 제공",
+    "VIEW_ONLY_RECENT": "최근의 주소를 선택",
+    "VIEW_ONLY_ENTER": "주소를 입력 (예. 0x4bbeEB066eD09...)",
+    "GO_TO_ACCOUNT": "계정으로 가기",
+    "INSECURE_WALLET_TYPE_TITLE": "이것은 지갑을 접근하기 위해 권장되는 방식이 아닙니다",
+    "INSECURE_WALLET_TYPE_DESC": "$wallet_type 형식의 지갑을 웹에서 사용하는 것은 위험합니다. 사이트가 손상되었거나, 실수로 피싱 사이트에 방문하는 경우 당신의 자금을 모두 잃어버릴 수 있습니다. 계속하기 전에 고려해보세요:",
```

Comment:
영어 원문     "INSECURE_WALLET_TYPE_DESC": "Entering your $wallet_type on any website is dangerous. If MyCrypto.com was compromised, or you accidentally visited a phishing website, you could lose your funds. Because of that, we have disabled the use of $wallet_type wallets through the website. In order to access your account, please download MyCrypto and run it locally.",

> 사이트가 손상되었거나

MyCrypto.com이 손상되었거나,

> 계속하기 전에 고려해보세요:",

이러한 이유때문에 저희는 웹 사이트에서 $wallet_type 형식의 지갑을 사용할 수 없게 해두었습니다. 계정에 접근하려면 MyCrypto 앱을 다운로드하여 로컬에서 실행하시길 바랍니다.